evap/static/scss/components/_tables.scss

```diff
@@ -156,3 +156,7 @@ tr.current-row-follows + tr {
         border: 1px solid $dark-gray;
     }
 }
+
+tr.odd-row {
+    background-color: darken($lighter-gray, 2%);
```

Comment:
it was an explicit decision to use our own color scheme (with more blue-ish gray) instead of the bootstrap color. we could also overwrite the bootstrap color, but i don't want to use it as is.

we should do something about your general dislike when generalizing this search box and applying it everywhere :)

Reply:
Um, but it already is overwritten? It differs in one percent, so I thought you actually wanted `3%`. And to avoid these errors, we got a variable :)

https://github.com/fsr-de/EvaP/blob/36ffa431a1f75f44cdb4a34b40ba556b358c0fe1/evap/static/scss/_variables.scss#L24

Well, the search box creates its own line adding much vertical space and the button looks very gray. But maybe it's just because it's new

Reply:
hm, the color looked strange somehow, but i'll check that again. i agree with the gray button - i don't like it too much as well, but didn't want to add another custom style yet. i'm happy to do so when deciding on a general search box solution.
and the vertical space is more or less the same as for other pages with search bar, so yes, probably because its new :)